src/main/window-manager.ts

```diff
@@ -214,9 +206,5 @@ export class WindowManager extends Singleton {
     this.splashWindow.destroy();
     this.mainWindow = null;
     this.splashWindow = null;
-    Object.entries(this.disposers).forEach(([name, dispose]) => {
-      dispose();
-      delete this.disposers[name];
-    });
```

Comment:
Should `WindowManager.destroy()` be private? It doesn't fully destroy anymore, being run as one of the disposers, so it's really only for internal use now.